web/src/containers/LoginPageRoutes.test.js

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import { renderWithConnection } from 'apd-testing-library';
+import LoginPageRoutes from './LoginPageRoutes';
+
+const props = {
+  hasEverLoggedOn: false,
+  fetching: false,
+  factorsList: [
+    {
+      id: 'emf5n20o64XfgZV7E297',
+      factorType: 'email',
+      provider: 'OKTA',
+      vendorName: 'OKTA',
+      profile: {
+        email: 'test@email.com'
+      }
+    }
+  ],
+  verifyData: {
+    qrcode: { href: 'www.test.com' }
+  },
+  handleFactorSelection: jest.fn(),
+  handlePhoneSubmit: jest.fn(),
+  handleVerificationCode: jest.fn(),
+  handleCreateAccessRequest: jest.fn(),
+  handleCompleteAccessRequest: jest.fn(),
+  handleLogin: jest.fn(),
+  handleLoginOtp: jest.fn(),
+  handleLogout: jest.fn()
+};
+
+describe('LoginPageRoutes', () => {
+  it('should redirect the user to Login if the path is /login', () => {
+    const useRouteMatch = jest.fn().mockReturnValue({ path: '/login' });
+    const { queryByRole } = renderWithConnection(
+      <LoginPageRoutes {...props} useRouteMatch={useRouteMatch} />,
+      {
+        initialHistory: ['/login']
+      }
+    );
+    expect(queryByRole('heading', { name: /Log in/i })).toBeTruthy();
+  });
+
+  it('should redirect the user to Account Locked if that path is /login/locked-out', () => {
+    const useRouteMatch = jest.fn().mockReturnValue({ path: '/login' });
+    const { queryByRole } = renderWithConnection(
+      <LoginPageRoutes {...props} useRouteMatch={useRouteMatch} />,
+      {
+        initialHistory: ['/login/locked-out']
+      }
+    );
+    expect(queryByRole('heading', { name: /Account Locked/i })).toBeTruthy();
+  });
+
+  it('should redirect the user to LoginMFAEnroll if that path is /login/mfa/enroll', () => {
```

Comment:
There seems to be a lot of repetition in this test.  I think we could shorten it up considerably if we implemented a table test.